pkg/loop/internal/telemetry.go

```diff
@@ -3,70 +3,56 @@ package internal
 import (
 	"context"
 	"errors"
-	"fmt"
 
-	"github.com/smartcontractkit/libocr/commontypes"
 	"google.golang.org/grpc"
 	"google.golang.org/protobuf/types/known/emptypb"
 
-	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
 	"github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb"
 	"github.com/smartcontractkit/chainlink-relay/pkg/types"
 )
 
-var _ types.Telemetry = (*telemetryClient)(nil)
-var _ types.MonitoringEndpointGenerator = (*telemetryClient)(nil)
-var _ commontypes.MonitoringEndpoint = (*telemetryEndpoint)(nil)
+var _ types.TelemetryClient = (*telemetryClient)(nil)
```

Comment:
Note: I relaxed the interfaces here and allowed the server and client-side interfaces to diverge.

I think the fact that we were validating inputs while having an interface which didn't allow errors to be returned to callers directly was making the API awkward, and opted to relax that.